sunpy/net/dataretriever/clients.py

```diff
@@ -18,6 +19,7 @@
 _Fido.register(NOAAIndicesClient, NOAAIndicesClient._can_handle_query)
 _Fido.register(NOAAPredictClient, NOAAPredictClient._can_handle_query)
 _Fido.register(RHESSIClient, RHESSIClient._can_handle_query)
+_Fido.register(KanzelhoheClient, KanzelhoheClient._can_handle_query)
```

Comment:
Same here, it would look better and easier to check if we leave a blank line as in the import so we can keep some order in here :)